src/command_modules/azure-cli-interactive/azclishell/layout.py

```diff
@@ -109,17 +109,22 @@ def get_lexers(main_lex, exam_lex, tool_lex):
     if not main_lex:
         return None, None, None
     lexer = None
-    if issubclass(main_lex, PromptLex):
-        lexer = main_lex
-    elif issubclass(main_lex, PygLex):
-        lexer = PygmentsLexer(main_lex)
+    if main_lex:
+        if issubclass(main_lex, PromptLex):
+            lexer = main_lex
+        elif issubclass(main_lex, PygLex):
+            lexer = PygmentsLexer(main_lex)
 
     if exam_lex:
         if issubclass(exam_lex, PygLex):
             exam_lex = PygmentsLexer(exam_lex)
+    else:
+        exam_lex = None
     if tool_lex:
         if issubclass(tool_lex, PygLex):
             tool_lex = PygmentsLexer(tool_lex)
+    else:
+        tool_lex = None
```

Comment:
What's the type of `tool_lex`? If the condition `tool_lex` is false, why must it be set to `None`?